node/pkg/vaa/structs.go

```diff
@@ -415,20 +425,100 @@ func (v *VAA) AddSignature(key *ecdsa.PrivateKey, index uint8) {
 	})
 }
 
+// NOTE: This function assumes that the caller has verified that the VAA is from the token bridge.
+func IsTransfer(payload []byte) bool {
+	if len(payload) > 0 {
+		if payload[0] == 1 || payload[0] == 3 {
+			return true
+		}
+	}
+
+	return false
+}
+
+func DecodeTransferPayloadHdr(payload []byte) (*TransferPayloadHdr, error) {
```

Comment:
I am curious where we plan to start calling DecodeTransferPayloadHdr, as this will have potentially more implications into how robust this parsing needs to be and what the failure conditions will be, but we can of course settle that once we bake these primitives out.

Reply:
This will be used in the chain governor to determine if the VAA should be limited. You can definitely throw rocks at it when we get there.